static/app.js

```diff
@@ -7,5 +7,21 @@
 
     const diff2htmlUi = new Diff2HtmlUI(targetElement, diffFiles, config);
     diff2htmlUi.draw();
+
+    registerViewToggleHandlers(targetElement);
   });
+
+  function registerViewToggleHandlers(root) {
+    const viewedToggles = root.querySelectorAll(".d2h-file-collapse-input");
+    for (const el of viewedToggles) {
+      el.addEventListener("change", scrollHeaderIntoView);
+    }
```

Comment:
```suggestion
    viewedToggles.forEach((el) => el.addEventListener("change", scrollHeaderIntoView));
```
Consider using ES6 foreach here.

Reply:
Thanks; here I prefer the for..of loop. I consider it more readable, as it puts the loop and the action on different lines of code. Also, I expect to add another `addEventListener` line there in a future PR: when the user toggles, `app.js` has to report the event to the extension so it can be saved. Does that sound alright?

Reply:
No problems, I understand your point of view.
It was only a suggestion. :)